metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -89,6 +80,14 @@
           table-name
           table-name))
 
+(defn granule-constraint-sql
+  "Returns the sql to define constraint on provider granule table"
+  [provider table-name]
+  (util/validate-table-name table-name)
+  (case (:small provider)
+    false (granule-constraint-sql-false table-name)
```

Comment:
Is it worth separating these into separate funcs? Why not keep the case statement here along with the sql query creation logic?

Reply:
No, I refactored this function and it no longer calls other functions.